src/types/sequence.rs

```diff
@@ -566,6 +566,7 @@ impl<'v> crate::PyTryFrom<'v> for PySequence {
 }
 
 #[cfg(test)]
+#[cfg_attr(not(feature = "gil-refs"), allow(deprecated))]
```

Comment:
In terms of why some tests have this blanket applied to `mod tests`, I did this for files where the GIL ref API is tested in some detail in the contained tests, mostly inside `src/types` directory. I think we want to keep the GIL Ref tests for coverage anyway, so my feelings were to just do this now as one way to keep churn down, and revisit these after 0.21 is pushed and we're heading towards 0.22.